assets/js/script.js

```diff
@@ -119,209 +102,214 @@ var randomQuestion;
-var resultsMsg = document.querySelector(".results-msg");
-var highScoresArray = JSON.parse(localStorage.getItem("High Scores"))|| [] ; //creates empty array for each score to be added to
+var result = document.querySelector('.result');
+var userInput = document.querySelector('.name-input'); //grabs input field
+var submitBtn = document.querySelector('.submit'); // grabs submit button
+var highScoreForm = document.getElementById('high-score-input'); //grabs entire form
+var highScoreList = document.getElementById('high-score-list'); //grabs empty ul
+var resultsMsg = document.querySelector('.results-msg');
+var highScoresArray = JSON.parse(localStorage.getItem('High Scores')) || []; //creates empty array for each score to be added to
 
 function gameEndMessage() {
-    if (score === 100) {
-        resultsMsg.textContent = "Perfect Score!";
-    }
-    if (score >= 70 && score < 100) {
-        resultsMsg.textContent = "Great Work!";
-    }
-    if (score >= 50 && score < 70) {
-        resultsMsg.textContent = "Keep It Up!"
-    }
-    if (score < 50) {
-        resultsMsg.textContent = "Better Luck Next Time"
-    }
+  if (score === 100) {
+    resultsMsg.textContent = 'Perfect Score!';
+  }
+  if (score >= 70 && score < 100) {
+    resultsMsg.textContent = 'Great Work!';
+  }
+  if (score >= 50 && score < 70) {
+    resultsMsg.textContent = 'Keep It Up!';
+  }
+  if (score < 50) {
+    resultsMsg.textContent = 'Better Luck Next Time';
+  }
 }
 
-var liMaker = function(text) {
-    console.log(text)
-    var li = document.createElement("li");
-    li.textContent = text.user + " " + text.score + "%";
-    highScoreList.appendChild(li);
+var liMaker = function (text) {
+  console.log(text);
+  var li = document.createElement('li');
+  li.textContent = text.user + ' ' + text.score + '%';
+  highScoreList.appendChild(li);
 };
 
-highScoreForm.addEventListener("submit", function(e) {
-    e.preventDefault();
+highScoreForm.addEventListener('submit', function (e) {
+  e.preventDefault();
 
-    highScoresArray.push({
-        user: userInput.value,
-        score: score
-    });
+  highScoresArray.push({
+    user: userInput.value,
+    score: score,
+  });
 
-    localStorage.setItem("High Scores", JSON.stringify(highScoresArray));
-    dataParse();
-    userInput.value = "";
+  localStorage.setItem('High Scores', JSON.stringify(highScoresArray));
+  dataParse();
+  userInput.value = '';
 });
 
 dataParse();
 
 function dataParse() {
-var data = JSON.parse(localStorage.getItem("High Scores")) || [];
+  var data = JSON.parse(localStorage.getItem('High Scores')) || [];
 
-highScoreList.innerHTML = "";
+  highScoreList.innerHTML = '';
 
-for(let i=0; i< data.length; i++) {
-    for(j=0; j<data.length; j++) {
-        if ( data[i].score > data[j].score){
-            var temp = data[i]
-            data[i] = data[j]
-            data[j] = temp
-        };
-    };
-};
+  for (let i = 0; i < data.length; i++) {
```

Comment:
Looks like you're sorting here? There are more efficient ways to sort (check out js sort functions)